src/Common/PagesLayout/Footer.jsx

```diff
@@ -1,25 +1,27 @@
-import styled from 'styled-components';
+import styled from "styled-components";
 
 /* Components ---------------------------*/
-import Inset from './Inset.jsx';
-import SiteLogo from './SiteLogo/SiteLogo.jsx';
+import Inset from "./Inset.jsx";
+import SiteLogo from "./SiteLogo/SiteLogo.jsx";
 
 const Footer = () => {
     return (
-        <FooterStyled className='Footer'>
+        <FooterStyled className="Footer">
             <Inset>
                 <div className="logo">
                     <SiteLogo />
                 </div>
                 <h2>
-                    StockPredicter :: Equity, Finance, and Investment :: San Diego, CA
+                    StockPredicter :: Equity, Finance, and Investment :: San
+                    Diego, CA
+                </h2>
```

Comment:
Please ignore whitespace and quoting diffs, that was from my local editor prefs.

You were missing a closing `</H2>` here.